test/models.json

```diff
@@ -2559,5 +2559,12 @@
     "source": "https://raw.githubusercontent.com/kosslab-kr/Tizen-NN-Runtime/master/Xor/xorGate.lite",
     "format": "TensorFlow Lite v3",
     "link":   "https://github.com/kosslab-kr/Tizen-NN-Runtime"
+  },
+  {
+    "type":   "openvino",
```

Comment:
To run `npm run test openvino` you will need to rebase to the latest `origin/master`. The code for running this was changed upstream.